sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -1250,6 +1301,9 @@ object Radar {
                 if (status == RadarStatus.SUCCESS || status == RadarStatus.ERROR_NOT_FOUND) {
                     RadarSettings.setTripOptions(context, null)
 
+                    // return to previous tracking options after trip
+                    locationManager.restartPreviousTrackingOptions();
+
                     // flush location update to generate events
                     locationManager.getLocation(null)
```

Comment:
@corypisano Saw a `TODO` above, should form an opinion about whether this is needed in both cases.